Python/legacy_tracing.c

```diff
@@ -18,6 +18,8 @@ typedef struct _PyLegacyEventHandler {
     int event;
 } _PyLegacyEventHandler;
 
+#define _PyLegacyEventHandler_CAST(op)  ((_PyLegacyEventHandler *)(op))
```

Comment:
You can update the code that I modified recently for UBsan around line 212..391 to use this new macro.

Reply:
I think it's done in this PR :')